docs/get-started/getting-started-workload-qualification.md

```diff
@@ -54,15 +54,75 @@ Since the two tools are only analyzing Spark event logs they do not have the det
 captured from a running Spark job.  However it is very convenient because you can run the tools on
 existing logs and do not need a GPU cluster to run the tools.
 
-## 2. Function `explainPotentialGpuPlan` 
+## 2. Get the Explain Output
+
+This allows running queries on the CPU and the plugin will evaluate the queries as if it was
+going to run on the GPU and tell you what would and wouldn't have been run on the GPU.
+There are two ways to run this, one is running with the plugin set to explain only mode and
```

Comment:
Nit: Ideally should be using "RAPIDS Accelerator" rather than "plugin" in user docs.  Applies to other places in the PR.